modules/blog/blog-home/blog-home.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef } from 'react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import Image from 'next/image';
+
+import { PostInterface } from '../../../pages/blog';
+import { dateFormatting } from '../../../utils/date-formatting';
+import {
+  tiltEffectSettings,
+  getTiltEffectValues,
+} from '../../../utils/get-tilt-effect-values';
+
+import style from './blog-home.module.scss';
+
+interface PostHomeTypes {
+  imgPosition: string;
+  post: PostInterface;
+  size: string;
+}
+
+const BlogHome = ({ size, imgPosition = 'right', post }: PostHomeTypes) => {
+  const [isTitleHovered, setIsTitleHovered] = useState(false);
+  const [rotateX, setRotateX] = useState(0);
+  const [rotateY, setRotateY] = useState(0);
+  const titleRef = useRef(null);
+
+  const imgSize = size === 'large' ? 592 : 400;
+
+  const onMouseMouse = (event: any) => {
+    setIsTitleHovered(true);
+    const { valueX, valueY } = getTiltEffectValues(titleRef, event);
+
+    setRotateX(valueX);
+    setRotateY(valueY);
```

Comment:
Let's try not to use `any` for types. You can also keep variable declarations at the top of the function scope and group calls beneath (when the order doesn't matter, and I don't think it does here).

```suggestion
  const onMouseMouse = (event: React.MouseEvent) => {
    const { valueX, valueY } = getTiltEffectValues(titleRef, event);
    
    setIsTitleHovered(true);
    setRotateX(valueX);
    setRotateY(valueY);
```